### DISCUSSION TITLE
RSS feed *from* my Forem posts

### DISCUSSION THREAD

**Discussion by ellativity** [forem/forem#16889] [Feature Requests]
I would love to be able to cross-post between Forems as easily as I can cross-post from other blogs to a Forem.

Please can I have an RSS feed of my Forem posts, so I can import it into another Forem? Thank you!

**Comment by jeremyf** [comment]
It looks as though `https://dev.to/feed/:username` kicks out an RSS feed of latest articles.  Does that address you feature request?
